### ISSUE TITLE
Weird "?" marks in all of my themes.

### ISSUE THREAD

**Issue by arafat-ar13** [JanDeDobbeleer/oh-my-posh2#234]
I have always wanted to use oh-my-posh as I am a PowerShell user and someone who just loves the terminal. I am on Windows 7 and so use ConEmu as my terminal as I don't have access to the Windows Terminal. I have been using posh-git to view my git information in PowerShell till now. One issue in particular has been keeping me from fully using oh-my-posh themes. These are some important info that might be required:

- Windows 7
- ConEmu version 19.10.12.0
- PowerShell Core 7.0.0
- PSReadLine 2.0.0-rc2
- posh-git version 1.0.0-beta4
- oh-my-posh version 2.0.412
- Pansies version 1.2.1
- I am using Fira Code indeed (which I installed from their original repo).
- I am using posh-git and importing it _before_ oh-my-posh
- I am using a beta version of PSReadLine as recommended in the README 
- I use Conda to manage my Python environments but I import posh-git and oh-my-posh _before_ Conda does anything to my prompt. So that couldn't be a problem.
- I have PowerLine installed but I have no idea if it's necessary or not. Please tell me to remove it if this might be the cause of my issue. I am keeping it for now until I am told to remove it.

Now, let's get to the issue. I can see git info alright and some other colors too but there are a lot of "?" marks depending of what theme I am using. This makes the prompt look bad.
For example this is Agnoster:
![Agnoster theme issue](https://i.ibb.co/ryHv0Kc/agnoster-issue.jpg)

And this is Honukai:
![Honukai theme issue](https://i.ibb.co/vX96cCL/honukai-issue.jpg)

Both show environment name successfully but some icons do not appear, instead they appear as "?" marks. This must be some encoding issue or something.
It will mean a lot if someone can point out the issue and help me fix this. I really need it. Thanks!

**Comment by njtrentacoste** [comment]
It looks like maybe you have a font which doesn't contain all the necessary Powerline Glyphs.  I switched to Cascadia Code PL which so far contains all the referenced Powerline Glyphs.  Have you installed the latest Cascadia Code font?  Make sure you install Cascadia Code PL.  I too was seeing ? in the prompt.  I'm running Windows 10 with Windows Terminal, so not exactly your situation, but that fixed my issue.

[Cascadia Font Repo](https://github.com/microsoft/cascadia-code/releases)

**Comment by cardin** [comment]
Your FiraCode font doesn't have some of the glyphs required. Try using FiraCode Nerd Font https://github.com/ryanoasis/nerd-fonts which appends a few glyph sets onto FiraCode.

**Comment by arafat-ar13** [comment]
I recently switched over to Cascadia Code PL and it is an amazing font. I found a little issue though, installing Cascadia Code PL from their official repo still doesn't now show all the glyphs properly (this might be fixed in the future). So what I did was install the font from [Nerd Fonts official website](https://www.nerdfonts.com/). This version of Cascadia Code fixed almost all the remaining glyphs. But sadly I do see one or two glyph that couldn't properly be rendered. I just live with that given that my terminal looks beautiful already.
